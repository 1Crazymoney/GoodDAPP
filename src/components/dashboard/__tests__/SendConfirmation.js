import React from 'react'
import { getWebRouterComponentWithMocks } from './__util__'
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

describe('SendConfirmation', () => {
  it('renders without errors', () => {
    const SendConfirmation = getWebRouterComponentWithMocks('../SendConfirmation')
    const tree = renderer.create(<SendConfirmation />)
    expect(tree.toJSON()).toBeTruthy()
  })

  it('matches snapshot', () => {
    const SendConfirmation = getWebRouterComponentWithMocks('../SendConfirmation')
    const component = renderer.create(<SendConfirmation />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
