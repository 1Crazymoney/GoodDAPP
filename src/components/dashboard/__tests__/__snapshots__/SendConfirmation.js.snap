// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendConfirmation matches snapshot 1`] = `
<div
  style={
    Object {
      "WebkitAlignItems": "stretch",
      "WebkitBoxAlign": "stretch",
      "WebkitBoxDirection": "normal",
      "WebkitBoxFlex": 1,
      "WebkitBoxOrient": "vertical",
      "WebkitFlexBasis": "0%",
      "WebkitFlexDirection": "column",
      "WebkitFlexGrow": 1,
      "WebkitFlexShrink": 1,
      "alignItems": "stretch",
      "borderBottomStyle": "solid",
      "borderBottomWidth": "0px",
      "borderLeftStyle": "solid",
      "borderLeftWidth": "0px",
      "borderRightStyle": "solid",
      "borderRightWidth": "0px",
      "borderTopStyle": "solid",
      "borderTopWidth": "0px",
      "boxSizing": "border-box",
      "display": "flex",
      "flexBasis": "0%",
      "flexDirection": "column",
      "flexGrow": 1,
      "flexShrink": 1,
      "marginBottom": "0px",
      "marginLeft": "0px",
      "marginRight": "0px",
      "marginTop": "0px",
      "minHeight": "0px",
      "minWidth": "0px",
      "msFlexAlign": "stretch",
      "msFlexDirection": "column",
      "msFlexNegative": 1,
      "msFlexPositive": 1,
      "msFlexPreferredSize": "0%",
      "paddingBottom": "0px",
      "paddingLeft": "0px",
      "paddingRight": "0px",
      "paddingTop": "0px",
      "position": "relative",
      "zIndex": 0,
    }
  }
>
  <div
    style={
      Object {
        "WebkitAlignItems": "stretch",
        "WebkitBoxAlign": "stretch",
        "WebkitBoxDirection": "normal",
        "WebkitBoxFlex": 1,
        "WebkitBoxOrient": "vertical",
        "WebkitFlexBasis": "0%",
        "WebkitFlexDirection": "column",
        "WebkitFlexGrow": 1,
        "WebkitFlexShrink": 1,
        "alignItems": "stretch",
        "borderBottomStyle": "solid",
        "borderBottomWidth": "0px",
        "borderLeftStyle": "solid",
        "borderLeftWidth": "0px",
        "borderRightStyle": "solid",
        "borderRightWidth": "0px",
        "borderTopStyle": "solid",
        "borderTopWidth": "0px",
        "boxSizing": "border-box",
        "display": "flex",
        "flexBasis": "0%",
        "flexDirection": "column",
        "flexGrow": 1,
        "flexShrink": 1,
        "marginBottom": "0px",
        "marginLeft": "0px",
        "marginRight": "0px",
        "marginTop": "0px",
        "minHeight": "0px",
        "minWidth": "0px",
        "msFlexAlign": "stretch",
        "msFlexDirection": "column",
        "msFlexNegative": 1,
        "msFlexPositive": 1,
        "msFlexPreferredSize": "0%",
        "paddingBottom": "0px",
        "paddingLeft": "0px",
        "paddingRight": "0px",
        "paddingTop": "0px",
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      style={
        Object {
          "WebkitAlignItems": "stretch",
          "WebkitBoxAlign": "stretch",
          "WebkitBoxDirection": "normal",
          "WebkitBoxFlex": 1,
          "WebkitBoxOrient": "horizontal",
          "WebkitBoxPack": "center",
          "WebkitFlexBasis": "0%",
          "WebkitFlexDirection": "row",
          "WebkitFlexGrow": 1,
          "WebkitFlexShrink": 1,
          "WebkitJustifyContent": "center",
          "alignItems": "stretch",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0px",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0px",
          "borderRightStyle": "solid",
          "borderRightWidth": "0px",
          "borderTopStyle": "solid",
          "borderTopWidth": "0px",
          "boxSizing": "border-box",
          "display": "flex",
          "flexBasis": "0%",
          "flexDirection": "row",
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "center",
          "marginBottom": "0px",
          "marginLeft": "0px",
          "marginRight": "0px",
          "marginTop": "0px",
          "minHeight": "0px",
          "minWidth": "0px",
          "msFlexAlign": "stretch",
          "msFlexDirection": "row",
          "msFlexNegative": 1,
          "msFlexPack": "center",
          "msFlexPositive": 1,
          "msFlexPreferredSize": "0%",
          "paddingBottom": "0px",
          "paddingLeft": "0px",
          "paddingRight": "0px",
          "paddingTop": "0px",
          "position": "relative",
          "zIndex": 0,
        }
      }
    >
      <div
        style={
          Object {
            "WebkitAlignItems": "stretch",
            "WebkitBoxAlign": "stretch",
            "WebkitBoxDirection": "normal",
            "WebkitBoxFlex": 1,
            "WebkitBoxOrient": "vertical",
            "WebkitBoxPack": "justify",
            "WebkitFlexBasis": "0%",
            "WebkitFlexDirection": "column",
            "WebkitFlexGrow": 1,
            "WebkitFlexShrink": 1,
            "WebkitJustifyContent": "space-between",
            "alignItems": "stretch",
            "borderBottomStyle": "solid",
            "borderBottomWidth": "0px",
            "borderLeftStyle": "solid",
            "borderLeftWidth": "0px",
            "borderRightStyle": "solid",
            "borderRightWidth": "0px",
            "borderTopStyle": "solid",
            "borderTopWidth": "0px",
            "boxSizing": "border-box",
            "display": "flex",
            "flexBasis": "0%",
            "flexDirection": "column",
            "flexGrow": 1,
            "flexShrink": 1,
            "justifyContent": "space-between",
            "marginBottom": "0px",
            "marginLeft": "0px",
            "marginRight": "0px",
            "marginTop": "0px",
            "minHeight": "0px",
            "minWidth": "0px",
            "msFlexAlign": "stretch",
            "msFlexDirection": "column",
            "msFlexNegative": 1,
            "msFlexPack": "justify",
            "msFlexPositive": 1,
            "msFlexPreferredSize": "0%",
            "paddingBottom": "1rem",
            "paddingLeft": "1rem",
            "paddingRight": "1rem",
            "paddingTop": "1rem",
            "position": "relative",
            "width": "100%",
            "zIndex": 0,
          }
        }
      >
        <div
          style={
            Object {
              "WebkitAlignItems": "stretch",
              "WebkitBoxAlign": "stretch",
              "WebkitBoxDirection": "normal",
              "WebkitBoxOrient": "vertical",
              "WebkitFlexBasis": "auto",
              "WebkitFlexDirection": "column",
              "WebkitFlexShrink": 0,
              "alignItems": "stretch",
              "backgroundColor": "rgba(238,238,239,1.00)",
              "borderBottomLeftRadius": "5px",
              "borderBottomRightRadius": "5px",
              "borderBottomStyle": "solid",
              "borderBottomWidth": "0px",
              "borderLeftStyle": "solid",
              "borderLeftWidth": "0px",
              "borderRightStyle": "solid",
              "borderRightWidth": "0px",
              "borderTopLeftRadius": "5px",
              "borderTopRightRadius": "5px",
              "borderTopStyle": "solid",
              "borderTopWidth": "0px",
              "boxSizing": "border-box",
              "display": "flex",
              "flexBasis": "auto",
              "flexDirection": "column",
              "flexShrink": 0,
              "marginBottom": "15px",
              "marginLeft": "0px",
              "marginRight": "0px",
              "marginTop": "0px",
              "minHeight": "0px",
              "minWidth": "0px",
              "msFlexAlign": "stretch",
              "msFlexDirection": "column",
              "msFlexNegative": 0,
              "msFlexPreferredSize": "auto",
              "paddingBottom": "15px",
              "paddingLeft": "10px",
              "paddingRight": "10px",
              "paddingTop": "15px",
              "position": "relative",
              "zIndex": 0,
            }
          }
        >
          <div
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "WebkitBoxDirection": "normal",
                "WebkitBoxOrient": "horizontal",
                "WebkitBoxPack": "justify",
                "WebkitFlexBasis": "auto",
                "WebkitFlexDirection": "row",
                "WebkitFlexShrink": 0,
                "WebkitJustifyContent": "space-between",
                "alignItems": "center",
                "borderBottomStyle": "solid",
                "borderBottomWidth": "0px",
                "borderLeftStyle": "solid",
                "borderLeftWidth": "0px",
                "borderRightStyle": "solid",
                "borderRightWidth": "0px",
                "borderTopStyle": "solid",
                "borderTopWidth": "0px",
                "boxSizing": "border-box",
                "display": "flex",
                "flexBasis": "auto",
                "flexDirection": "row",
                "flexShrink": 0,
                "justifyContent": "space-between",
                "marginBottom": "0px",
                "marginLeft": "0px",
                "marginRight": "0px",
                "marginTop": "0px",
                "minHeight": "0px",
                "minWidth": "0px",
                "msFlexAlign": "center",
                "msFlexDirection": "row",
                "msFlexNegative": 0,
                "msFlexPack": "justify",
                "msFlexPreferredSize": "auto",
                "paddingBottom": "0px",
                "paddingLeft": "0px",
                "paddingRight": "0px",
                "paddingTop": "0px",
                "position": "relative",
                "zIndex": 0,
              }
            }
          >
            <div
              style={
                Object {
                  "WebkitAlignItems": "stretch",
                  "WebkitBoxAlign": "stretch",
                  "WebkitBoxDirection": "normal",
                  "WebkitBoxOrient": "vertical",
                  "WebkitFlexBasis": "auto",
                  "WebkitFlexDirection": "column",
                  "WebkitFlexShrink": 0,
                  "alignItems": "stretch",
                  "backgroundColor": "rgba(255,255,255,1.00)",
                  "borderBottomLeftRadius": "17px",
                  "borderBottomRightRadius": "17px",
                  "borderBottomStyle": "solid",
                  "borderBottomWidth": "0px",
                  "borderLeftStyle": "solid",
                  "borderLeftWidth": "0px",
                  "borderRightStyle": "solid",
                  "borderRightWidth": "0px",
                  "borderTopLeftRadius": "17px",
                  "borderTopRightRadius": "17px",
                  "borderTopStyle": "solid",
                  "borderTopWidth": "0px",
                  "boxSizing": "border-box",
                  "display": "flex",
                  "flexBasis": "auto",
                  "flexDirection": "column",
                  "flexShrink": 0,
                  "height": "34px",
                  "marginBottom": "0px",
                  "marginLeft": "0px",
                  "marginRight": "0px",
                  "marginTop": "0px",
                  "minHeight": "0px",
                  "minWidth": "0px",
                  "msFlexAlign": "stretch",
                  "msFlexDirection": "column",
                  "msFlexNegative": 0,
                  "msFlexPreferredSize": "auto",
                  "paddingBottom": "0px",
                  "paddingLeft": "0px",
                  "paddingRight": "0px",
                  "paddingTop": "0px",
                  "position": "relative",
                  "width": "34px",
                  "zIndex": 0,
                }
              }
            >
              <div
                style={
                  Object {
                    "WebkitAlignItems": "stretch",
                    "WebkitBoxAlign": "stretch",
                    "WebkitBoxDirection": "normal",
                    "WebkitBoxOrient": "vertical",
                    "WebkitFlexBasis": "auto",
                    "WebkitFlexDirection": "column",
                    "WebkitFlexShrink": 0,
                    "alignItems": "stretch",
                    "borderBottomLeftRadius": "17px",
                    "borderBottomRightRadius": "17px",
                    "borderBottomStyle": "solid",
                    "borderBottomWidth": "0px",
                    "borderLeftStyle": "solid",
                    "borderLeftWidth": "0px",
                    "borderRightStyle": "solid",
                    "borderRightWidth": "0px",
                    "borderTopLeftRadius": "17px",
                    "borderTopRightRadius": "17px",
                    "borderTopStyle": "solid",
                    "borderTopWidth": "0px",
                    "boxSizing": "border-box",
                    "display": "flex",
                    "flexBasis": "auto",
                    "flexDirection": "column",
                    "flexShrink": 0,
                    "height": "34px",
                    "marginBottom": "0px",
                    "marginLeft": "0px",
                    "marginRight": "0px",
                    "marginTop": "0px",
                    "minHeight": "0px",
                    "minWidth": "0px",
                    "msFlexAlign": "stretch",
                    "msFlexDirection": "column",
                    "msFlexNegative": 0,
                    "msFlexPreferredSize": "auto",
                    "overflowX": "hidden",
                    "overflowY": "hidden",
                    "paddingBottom": "0px",
                    "paddingLeft": "0px",
                    "paddingRight": "0px",
                    "paddingTop": "0px",
                    "position": "relative",
                    "width": "34px",
                    "zIndex": 0,
                  }
                }
              >
                <div
                  style={
                    Object {
                      "WebkitAlignItems": "stretch",
                      "WebkitBoxAlign": "stretch",
                      "WebkitBoxDirection": "normal",
                      "WebkitBoxOrient": "vertical",
                      "WebkitFlexBasis": "auto",
                      "WebkitFlexDirection": "column",
                      "WebkitFlexShrink": 0,
                      "alignItems": "stretch",
                      "backgroundColor": "rgba(0,0,0,0.00)",
                      "backgroundPosition": "center",
                      "backgroundRepeat": "no-repeat",
                      "backgroundSize": "cover",
                      "borderBottomStyle": "solid",
                      "borderBottomWidth": "0px",
                      "borderLeftStyle": "solid",
                      "borderLeftWidth": "0px",
                      "borderRightStyle": "solid",
                      "borderRightWidth": "0px",
                      "borderTopStyle": "solid",
                      "borderTopWidth": "0px",
                      "bottom": "0px",
                      "boxSizing": "border-box",
                      "display": "flex",
                      "flexBasis": "auto",
                      "flexDirection": "column",
                      "flexShrink": 0,
                      "height": "100%",
                      "left": "0px",
                      "marginBottom": "0px",
                      "marginLeft": "0px",
                      "marginRight": "0px",
                      "marginTop": "0px",
                      "minHeight": "0px",
                      "minWidth": "0px",
                      "msFlexAlign": "stretch",
                      "msFlexDirection": "column",
                      "msFlexNegative": 0,
                      "msFlexPreferredSize": "auto",
                      "paddingBottom": "0px",
                      "paddingLeft": "0px",
                      "paddingRight": "0px",
                      "paddingTop": "0px",
                      "position": "absolute",
                      "right": "0px",
                      "top": "0px",
                      "width": "100%",
                      "zIndex": -1,
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          style={
            Object {
              "WebkitAlignItems": "center",
              "WebkitBoxAlign": "center",
              "WebkitBoxDirection": "normal",
              "WebkitBoxFlex": 1,
              "WebkitBoxOrient": "vertical",
              "WebkitFlexBasis": "0%",
              "WebkitFlexDirection": "column",
              "WebkitFlexGrow": 1,
              "WebkitFlexShrink": 1,
              "alignItems": "center",
              "backgroundColor": "rgba(238,238,239,1.00)",
              "borderBottomLeftRadius": "5px",
              "borderBottomRightRadius": "5px",
              "borderBottomStyle": "solid",
              "borderBottomWidth": "0px",
              "borderLeftStyle": "solid",
              "borderLeftWidth": "0px",
              "borderRightStyle": "solid",
              "borderRightWidth": "0px",
              "borderTopLeftRadius": "5px",
              "borderTopRightRadius": "5px",
              "borderTopStyle": "solid",
              "borderTopWidth": "0px",
              "boxSizing": "border-box",
              "display": "flex",
              "flexBasis": "0%",
              "flexDirection": "column",
              "flexGrow": 1,
              "flexShrink": 1,
              "marginBottom": "15px",
              "marginLeft": "0px",
              "marginRight": "0px",
              "marginTop": "0px",
              "minHeight": "0px",
              "minWidth": "0px",
              "msFlexAlign": "center",
              "msFlexDirection": "column",
              "msFlexNegative": 1,
              "msFlexPositive": 1,
              "msFlexPreferredSize": "0%",
              "paddingBottom": "15px",
              "paddingLeft": "10px",
              "paddingRight": "10px",
              "paddingTop": "15px",
              "position": "relative",
              "zIndex": 0,
            }
          }
        >
          <div
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "WebkitBoxDirection": "normal",
                "WebkitBoxOrient": "horizontal",
                "WebkitBoxPack": "justify",
                "WebkitFlexBasis": "auto",
                "WebkitFlexDirection": "row",
                "WebkitFlexShrink": 0,
                "WebkitJustifyContent": "space-between",
                "alignItems": "center",
                "borderBottomStyle": "solid",
                "borderBottomWidth": "0px",
                "borderLeftStyle": "solid",
                "borderLeftWidth": "0px",
                "borderRightStyle": "solid",
                "borderRightWidth": "0px",
                "borderTopStyle": "solid",
                "borderTopWidth": "0px",
                "boxSizing": "border-box",
                "display": "flex",
                "flexBasis": "auto",
                "flexDirection": "row",
                "flexShrink": 0,
                "justifyContent": "space-between",
                "marginBottom": "0px",
                "marginLeft": "0px",
                "marginRight": "0px",
                "marginTop": "0px",
                "minHeight": "0px",
                "minWidth": "0px",
                "msFlexAlign": "center",
                "msFlexDirection": "row",
                "msFlexNegative": 0,
                "msFlexPack": "justify",
                "msFlexPreferredSize": "auto",
                "paddingBottom": "0px",
                "paddingLeft": "0px",
                "paddingRight": "0px",
                "paddingTop": "0px",
                "position": "relative",
                "zIndex": 0,
              }
            }
          >
            <div
              style={
                Object {
                  "WebkitAlignItems": "stretch",
                  "WebkitBoxAlign": "stretch",
                  "WebkitBoxDirection": "normal",
                  "WebkitBoxOrient": "vertical",
                  "WebkitFlexBasis": "auto",
                  "WebkitFlexDirection": "column",
                  "WebkitFlexShrink": 0,
                  "alignItems": "stretch",
                  "borderBottomColor": "rgba(85,85,85,1.00)",
                  "borderBottomLeftRadius": "4px",
                  "borderBottomRightRadius": "4px",
                  "borderBottomStyle": "solid",
                  "borderBottomWidth": "1px",
                  "borderLeftColor": "rgba(85,85,85,1.00)",
                  "borderLeftStyle": "solid",
                  "borderLeftWidth": "1px",
                  "borderRightColor": "rgba(85,85,85,1.00)",
                  "borderRightStyle": "solid",
                  "borderRightWidth": "1px",
                  "borderTopColor": "rgba(85,85,85,1.00)",
                  "borderTopLeftRadius": "4px",
                  "borderTopRightRadius": "4px",
                  "borderTopStyle": "solid",
                  "borderTopWidth": "1px",
                  "boxSizing": "border-box",
                  "display": "flex",
                  "flexBasis": "auto",
                  "flexDirection": "column",
                  "flexShrink": 0,
                  "marginBottom": "0px",
                  "marginLeft": "0px",
                  "marginRight": "0px",
                  "marginTop": "2rem",
                  "minHeight": "0px",
                  "minWidth": "0px",
                  "msFlexAlign": "stretch",
                  "msFlexDirection": "column",
                  "msFlexNegative": 0,
                  "msFlexPreferredSize": "auto",
                  "paddingBottom": "1rem",
                  "paddingLeft": "1rem",
                  "paddingRight": "1rem",
                  "paddingTop": "1rem",
                  "position": "relative",
                  "zIndex": 0,
                }
              }
            >
              <canvas
                height={128}
                style={
                  Object {
                    "height": 128,
                    "width": 128,
                  }
                }
                width={128}
              />
            </div>
          </div>
          <div
            style={
              Object {
                "WebkitAlignItems": "stretch",
                "WebkitBoxAlign": "stretch",
                "WebkitBoxDirection": "normal",
                "WebkitBoxOrient": "vertical",
                "WebkitFlexBasis": "auto",
                "WebkitFlexDirection": "column",
                "WebkitFlexShrink": 0,
                "alignItems": "stretch",
                "borderBottomStyle": "solid",
                "borderBottomWidth": "0px",
                "borderLeftStyle": "solid",
                "borderLeftWidth": "0px",
                "borderRightStyle": "solid",
                "borderRightWidth": "0px",
                "borderTopStyle": "solid",
                "borderTopWidth": "0px",
                "boxSizing": "border-box",
                "display": "flex",
                "flexBasis": "auto",
                "flexDirection": "column",
                "flexShrink": 0,
                "marginBottom": "1rem",
                "marginLeft": "0px",
                "marginRight": "0px",
                "marginTop": "0px",
                "minHeight": "0px",
                "minWidth": "0px",
                "msFlexAlign": "stretch",
                "msFlexDirection": "column",
                "msFlexNegative": 0,
                "msFlexPreferredSize": "auto",
                "paddingBottom": "0px",
                "paddingLeft": "0px",
                "paddingRight": "0px",
                "paddingTop": "0px",
                "position": "relative",
                "zIndex": 0,
              }
            }
          >
            <div
              dir="auto"
              style={
                Object {
                  "borderBottomWidth": "0px",
                  "borderLeftWidth": "0px",
                  "borderRightWidth": "0px",
                  "borderTopWidth": "0px",
                  "boxSizing": "border-box",
                  "color": "rgba(85,85,85,1.00)",
                  "display": "inline",
                  "fontFamily": "Helvetica, \\"sans-serif\\"",
                  "fontSize": "18px",
                  "fontStyle": "inherit",
                  "fontVariant": "inherit",
                  "fontWeight": "inherit",
                  "lineHeight": "inherit",
                  "marginBottom": "0px",
                  "marginLeft": "0px",
                  "marginRight": "0px",
                  "marginTop": "0px",
                  "paddingBottom": "0px",
                  "paddingLeft": "0px",
                  "paddingRight": "0px",
                  "paddingTop": "0px",
                  "textAlign": "center",
                  "textDecoration": "none",
                  "whiteSpace": "pre-wrap",
                  "wordWrap": "break-word",
                }
              }
            >
              <span
                dir="auto"
                style={
                  Object {
                    "borderBottomWidth": "0px",
                    "borderLeftWidth": "0px",
                    "borderRightWidth": "0px",
                    "borderTopWidth": "0px",
                    "boxSizing": "border-box",
                    "color": "inherit",
                    "display": "inline",
                    "fontFamily": "inherit",
                    "fontSize": "inherit",
                    "fontStyle": "inherit",
                    "fontVariant": "inherit",
                    "fontWeight": "inherit",
                    "lineHeight": "inherit",
                    "marginBottom": "0px",
                    "marginLeft": "0px",
                    "marginRight": "0px",
                    "marginTop": "0px",
                    "paddingBottom": "0px",
                    "paddingLeft": "0px",
                    "paddingRight": "0px",
                    "paddingTop": "0px",
                    "textDecoration": "none",
                    "whiteSpace": "inherit",
                    "wordWrap": "break-word",
                  }
                }
              />
            </div>
            <div
              data-focusable={true}
              dir="auto"
              onClick={[Function]}
              onKeyDown={[Function]}
              style={
                Object {
                  "borderBottomWidth": "0px",
                  "borderLeftWidth": "0px",
                  "borderRightWidth": "0px",
                  "borderTopWidth": "0px",
                  "boxSizing": "border-box",
                  "color": "rgba(162,162,162,1.00)",
                  "cursor": "pointer",
                  "display": "inline",
                  "fontFamily": "Helvetica, \\"sans-serif\\"",
                  "fontSize": "14px",
                  "fontStyle": "inherit",
                  "fontVariant": "inherit",
                  "fontWeight": "inherit",
                  "lineHeight": "inherit",
                  "marginBottom": "1rem",
                  "marginLeft": "1rem",
                  "marginRight": "1rem",
                  "marginTop": "1rem",
                  "paddingBottom": "0px",
                  "paddingLeft": "0px",
                  "paddingRight": "0px",
                  "paddingTop": "0px",
                  "textAlign": "center",
                  "textDecoration": "none",
                  "textTransform": "uppercase",
                  "whiteSpace": "pre-wrap",
                  "wordWrap": "break-word",
                }
              }
              tabIndex="0"
            >
              Copy address to clipboard
            </div>
          </div>
          <div
            style={
              Object {
                "WebkitAlignItems": "stretch",
                "WebkitBoxAlign": "stretch",
                "WebkitBoxDirection": "normal",
                "WebkitBoxOrient": "vertical",
                "WebkitBoxPack": "center",
                "WebkitFlexBasis": "auto",
                "WebkitFlexDirection": "column",
                "WebkitFlexShrink": 0,
                "WebkitJustifyContent": "center",
                "alignItems": "stretch",
                "backgroundColor": "rgba(0,0,0,0.00)",
                "borderBottomColor": "rgba(0,0,0,0.00)",
                "borderBottomLeftRadius": "4px",
                "borderBottomRightRadius": "4px",
                "borderBottomStyle": "solid",
                "borderBottomWidth": "0px",
                "borderLeftColor": "rgba(0,0,0,0.00)",
                "borderLeftStyle": "solid",
                "borderLeftWidth": "0px",
                "borderRightColor": "rgba(0,0,0,0.00)",
                "borderRightStyle": "solid",
                "borderRightWidth": "0px",
                "borderTopColor": "rgba(0,0,0,0.00)",
                "borderTopLeftRadius": "4px",
                "borderTopRightRadius": "4px",
                "borderTopStyle": "solid",
                "borderTopWidth": "0px",
                "boxShadow": "0px 0px 0px rgba(0,0,0,0.24)",
                "boxSizing": "border-box",
                "display": "flex",
                "flexBasis": "auto",
                "flexDirection": "column",
                "flexShrink": 0,
                "justifyContent": "center",
                "marginBottom": "0px",
                "marginLeft": "0px",
                "marginRight": "0px",
                "marginTop": "0px",
                "minHeight": "0px",
                "minWidth": "64px",
                "msFlexAlign": "stretch",
                "msFlexDirection": "column",
                "msFlexNegative": 0,
                "msFlexPack": "center",
                "msFlexPreferredSize": "auto",
                "paddingBottom": "0px",
                "paddingLeft": "0px",
                "paddingRight": "0px",
                "paddingTop": "0px",
                "position": "relative",
                "zIndex": 0,
              }
            }
          >
            <div
              data-focusable={true}
              disabled={false}
              onKeyDown={[Function]}
              onKeyPress={[Function]}
              onKeyUp={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              role="button"
              style={
                Object {
                  "WebkitAlignItems": "stretch",
                  "WebkitBoxAlign": "stretch",
                  "WebkitBoxDirection": "normal",
                  "WebkitBoxOrient": "vertical",
                  "WebkitFlexBasis": "auto",
                  "WebkitFlexDirection": "column",
                  "WebkitFlexShrink": 0,
                  "alignItems": "stretch",
                  "borderBottomLeftRadius": "4px",
                  "borderBottomRightRadius": "4px",
                  "borderBottomStyle": "solid",
                  "borderBottomWidth": "0px",
                  "borderLeftStyle": "solid",
                  "borderLeftWidth": "0px",
                  "borderRightStyle": "solid",
                  "borderRightWidth": "0px",
                  "borderTopLeftRadius": "4px",
                  "borderTopRightRadius": "4px",
                  "borderTopStyle": "solid",
                  "borderTopWidth": "0px",
                  "boxSizing": "border-box",
                  "cursor": "pointer",
                  "display": "flex",
                  "flexBasis": "auto",
                  "flexDirection": "column",
                  "flexShrink": 0,
                  "marginBottom": "0px",
                  "marginLeft": "0px",
                  "marginRight": "0px",
                  "marginTop": "0px",
                  "minHeight": "0px",
                  "minWidth": "0px",
                  "msFlexAlign": "stretch",
                  "msFlexDirection": "column",
                  "msFlexNegative": 0,
                  "msFlexPreferredSize": "auto",
                  "msTouchAction": "manipulation",
                  "overflowX": "hidden",
                  "overflowY": "hidden",
                  "paddingBottom": "0px",
                  "paddingLeft": "0px",
                  "paddingRight": "0px",
                  "paddingTop": "0px",
                  "position": "relative",
                  "touchAction": "manipulation",
                  "zIndex": 0,
                }
              }
              tabIndex="0"
            >
              <div
                style={
                  Object {
                    "WebkitAlignItems": "center",
                    "WebkitBoxAlign": "center",
                    "WebkitBoxDirection": "normal",
                    "WebkitBoxOrient": "horizontal",
                    "WebkitBoxPack": "center",
                    "WebkitFlexBasis": "auto",
                    "WebkitFlexDirection": "row",
                    "WebkitFlexShrink": 0,
                    "WebkitJustifyContent": "center",
                    "alignItems": "center",
                    "borderBottomStyle": "solid",
                    "borderBottomWidth": "0px",
                    "borderLeftStyle": "solid",
                    "borderLeftWidth": "0px",
                    "borderRightStyle": "solid",
                    "borderRightWidth": "0px",
                    "borderTopStyle": "solid",
                    "borderTopWidth": "0px",
                    "boxSizing": "border-box",
                    "display": "flex",
                    "flexBasis": "auto",
                    "flexDirection": "row",
                    "flexShrink": 0,
                    "justifyContent": "center",
                    "marginBottom": "0px",
                    "marginLeft": "0px",
                    "marginRight": "0px",
                    "marginTop": "0px",
                    "minHeight": "0px",
                    "minWidth": "0px",
                    "msFlexAlign": "center",
                    "msFlexDirection": "row",
                    "msFlexNegative": 0,
                    "msFlexPack": "center",
                    "msFlexPreferredSize": "auto",
                    "paddingBottom": "0px",
                    "paddingLeft": "0px",
                    "paddingRight": "0px",
                    "paddingTop": "0px",
                    "position": "relative",
                    "zIndex": 0,
                  }
                }
              >
                <div
                  dir="auto"
                  style={
                    Object {
                      "borderBottomWidth": "0px",
                      "borderLeftWidth": "0px",
                      "borderRightWidth": "0px",
                      "borderTopWidth": "0px",
                      "boxSizing": "border-box",
                      "color": "rgba(85,85,85,1.00)",
                      "direction": "ltr",
                      "display": "inline",
                      "fontFamily": "Roboto-Medium, Roboto, \\"Helvetica Neue\\", Helvetica, Arial, sans-serif",
                      "fontSize": "14px",
                      "fontStyle": "inherit",
                      "fontVariant": "inherit",
                      "fontWeight": "inherit",
                      "letterSpacing": "1px",
                      "lineHeight": "inherit",
                      "marginBottom": "9px",
                      "marginLeft": "16px",
                      "marginRight": "16px",
                      "marginTop": "9px",
                      "maxWidth": "100%",
                      "overflowX": "hidden",
                      "overflowY": "hidden",
                      "paddingBottom": "0px",
                      "paddingLeft": "0px",
                      "paddingRight": "0px",
                      "paddingTop": "0px",
                      "textAlign": "center",
                      "textDecoration": "none",
                      "textOverflow": "ellipsis",
                      "whiteSpace": "nowrap",
                      "wordWrap": "break-word",
                    }
                  }
                >
                  <span
                    dir="auto"
                    style={
                      Object {
                        "borderBottomWidth": "0px",
                        "borderLeftWidth": "0px",
                        "borderRightWidth": "0px",
                        "borderTopWidth": "0px",
                        "boxSizing": "border-box",
                        "direction": "ltr",
                        "display": "inline",
                        "fontFamily": "Helvetica, \\"sans-serif\\"",
                        "fontSize": "18px",
                        "fontStyle": "inherit",
                        "fontVariant": "inherit",
                        "fontWeight": "bold",
                        "lineHeight": "inherit",
                        "marginBottom": "0px",
                        "marginLeft": "0px",
                        "marginRight": "0px",
                        "marginTop": "0px",
                        "paddingBottom": "5px",
                        "paddingLeft": "5px",
                        "paddingRight": "5px",
                        "paddingTop": "5px",
                        "textAlign": "center",
                        "textDecoration": "none",
                        "textTransform": "uppercase",
                        "whiteSpace": "inherit",
                        "wordWrap": "break-word",
                      }
                    }
                  >
                    Share Link
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
